xds/internal/balancer/balancergroup/balancergroup.go

```diff
@@ -35,16 +35,16 @@ import (
 	"google.golang.org/grpc/xds/internal/balancer/lrs"
 )
 
-// subBalancerWithConfig is used to keep the configurations that will be used to start
+// subBalanceWrapper is used to keep the configurations that will be used to start
 // the underlying balancer. It can be called to start/stop the underlying
 // balancer.
 //
 // When the config changes, it will pass the update to the underlying balancer
 // if it exists.
 //
-// TODO: rename to subBalanceWrapper (and move to a separate file?)
```

Comment:
Oh no, this was a typo!  It was supposed to be renamed to subBalance**r**Wrapper.  Sorry!!